pilot/pkg/security/authz/builder/v1.go

```diff
@@ -0,0 +1,82 @@
+// Copyright 2019 Istio Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package builder
+
+import (
+	http_config "github.com/envoyproxy/go-control-plane/envoy/config/filter/http/rbac/v2"
+	envoy_rbac "github.com/envoyproxy/go-control-plane/envoy/config/rbac/v2"
+
+	istio_rbac "istio.io/api/rbac/v1alpha1"
+)
+
+// buildV1 builds the v1 policy to filter config.
+func (b *Builder) buildV1(forTCPFilter bool) *http_config.RBAC {
```

Comment:
how about moving version specific files to subfolder as authn does ?

Reply:
Done